plugins/GSdx/GSLocalMemory.cpp

```diff
@@ -41,7 +41,7 @@
 		for(int x = _r.left; x < _r.right; x += w >> 3) \
 		{ \
 			const uint8* src = BlockPtr(_base + off->block.col[x]); \
-			uint8* dst = &_dst[x * bpp]; \
+			dst = &_dst[x * bpp]; \
```

Comment:
isn't _dst pointing to a different position to dst though? I thought the point in this function was to move data to a different location in memory.  If we initialize a pointer in _dst it should be independent of dst, however when this function is called "dst" already exists, we were hiding the previous declaration.